### PR TITLE
Create markdown script

### PR DESCRIPTION
**Pull request by ibramoha**

### CODE REVIEW

File: script/data.js [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-  "Families",
-  "Gender Based Violence",
-  "Mental Health Services",
-  "Social and Other",
-  "Women",
-  "Pregnant Women and New Mothers",
-  "Baby Equipment",
-  "Young People/Children"
-];
+// const gDriveURL =
```
Comment (kkarimi) [inline script/data.js]: if you think it's a good idea to leave commented code in the repo, at least put a comment to explain why you are keeping it.

File: script/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "data.js",
   "scripts": {
-    "start": "node data.js",
+    "start": "node data.js; node addLatLong.js",
```
Comment (kkarimi) [inline script/package.json]: Where is this process documented? and why can't one of these files call the other one?

File: script/data.js (hunk 1 above)
Comment (ibramoha) [reply script/data.js]: Alright @kkarimi I will add the comment.

File: script/package.json (hunk 2 above)
Comment (ibramoha) [reply script/package.json]: We forgot to put it on the doc,I will add it. we have tried to call one file from the other but it get complicated, But we will do it that way if it is a good practice.

**Pull request merged**